charts/otel-cloud-stack/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: otel-cloud-stack
```

Comment:
Whats the origin for this name.  If the goal is to be like `kube-prometheus-stack` should we use `kube-otel-stack`? Or to stick with our existing naming convention, should we do `opentelemetry-kube-stack`?

Reply:
@TylerHelmuth im fine with anything to be honest. I originally made kube-otel-stack, and then i created a new chart that was entirely otel-based and wanted something differentiating so i did otel-cloud-stack. I'm fine with whatever you all think is best.

Reply:
I vote goes to `opentelemetry-kube-stack` to match our existing naming convention.

Reply:
Kube otel stack is more synonymous to kube prom stack. ( it also does much more than just metrics) But following naming conventions is more important.

Reply:
kube-otel-stack or otel-kube-stack

Reply:
`opentelemetry-kube-stack` sounds good to me

Reply:
@povilasv @puckpuck @JaredTan95 please share your opinion.

Reply:
that sounds good to me... changing now.